scripts/dependency-check/create-pull-request

```diff
@@ -86,7 +86,7 @@ const OWNER = 'contentful';
   const { data: pullRequests } = await githubClient.pulls.list({ owner: OWNER, repo: REPOSITORY, state: 'open' });
 
   for (const pullRequest of pullRequests) {
-    if (pullRequest.title === PULL_REQUEST_TITLE) {
+    if (pullRequest.title === PULL_REQUEST_TITLE && pullRequest.number !== newPullRequest.number) {
```

Comment:
Why not use the number only? It should be a unique identifier on its own.

Reply:
Because we want to close all the PRs with the same title and different number, not all the PRs in general